app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="post">
+  <div class="title-info">
+    <h2>Post #<%= @post.id%></h2>
```

Comment:
- Please note that after adding populating the database with some data, I get the below error when I start the server 👇🏽.
Kindly fix this.
![Screenshot from 2023-02-16 16-09-49](https://user-images.githubusercontent.com/44624138/219410205-1bf75d03-db3a-4019-9c2d-a24162bf4165.png)

Reply:
hello @rotimiazeez thank you for your review please any idea or hint on how I can fix this please I am actually struggling with rails..thank you

Reply:
Hi @Ngufor-emmanuella,
The error NoMethodError means that you call a method that doesn't exist for the given class. In this case, it is nil: NilClass, which doesn't contain the method each because it is nil! So let's examine:

You are calling @post.id, which is calling the error. Since it says that the method is undefined for nil (as described above), it tells us that on the index page, @post.id is nil (probably not what it is intended to be!).

So the problem rests in your app/views/index.html.erb, I'm not entirely sure, but it looks as though. Kindly try using the code below in your app/views/index.html.erb because, at the moment, it seems you have the wrong info in there at the moment👇🏽
```
<% @users.each do |person| %>
  <div class="wrapper flex w80 mAuto">
    <div>
      <%= image_tag(person.photo, :class => 'photo border')%>
    </div>
    <%= link_to user_path(:id => person.id), class: 'anchor_user' do%>
    <div class="user_info w100 border flex">
      <h2 class="w50"><%= person.name%></h2>
      <p class="w50 postNum">Number of posts: <%= person.posts_ounter%></p>
    </div>
    <% end %>
  </div>
<% end %>
```

Reply:
thank you so much